models/asset.js

```diff
@@ -0,0 +1,110 @@
+const mongoose = require('../mongoose');
+const uuid = require('uuid');
+const Schema = mongoose.Schema;
+
+const AssetSchema = new Schema({
+
+  id: {
+    type: String,
+    default: uuid.v4,
+    unique: true,
+    index: true
+  },
+  url: {
+    type: String,
+    unique: true,
+    index: true
+  },
+  type: {
+    type: String,
+    default: 'article'
+  },
+  headline: String,
+  summary: String,
+  section: String,
+  subsection: String,
+  authors: [String],
+  publication_date: Date
+},{
+  _id: false,
```

Comment:
Should not use `_id: false` here, Mongoose needs is for root documents in a collection :(